spec/system/avo/discreet_pagination_spec.rb

```diff
@@ -0,0 +1,32 @@
+require "rails_helper"
+
+RSpec.describe "DiscreetPagination", type: :system do
```

Comment:
Can we make this a `feature` test? This will make it way faster.
Sorry I haven't mentioned it before.

System tests will not work with this approach because Turbo is loading the frame in the browser.
In order to test it as a `feature` test, we should load the actual frame and test our assertion there. Something similar to [this](https://github.com/avo-hq/avo/blob/main/spec/requests/avo/authorization/associations_spec.rb#L14).

Reply:
Sure! System type took 4 seconds on this test and feature type only 1 second, great improvement!